app/main/movie_data_service.py

```diff
@@ -0,0 +1,22 @@
+from app.main.models import Movie
+
+
+class MovieDataService:
```

Comment:
I really like it :) but please add description in doc string what for is this class